roxctl/common/npguardutils/netpolutils.go

```diff
@@ -0,0 +1,36 @@
+package npguardutils
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+
+	e "github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/errox"
+)
+
+var (
+	// ErrNPGErrorsIndicator errors indicator message
+	ErrNPGErrorsIndicator = errors.New("there were errors during execution")
+	// ErrNPGWarningsIndicator warnings indicator message
+	ErrNPGWarningsIndicator = errors.New("there were warnings during execution")
+)
+
+// WriteFile writes output file for the relevant NP-Guard command
+func WriteFile(filename string, destDir string, content string) error {
+	outputPath := filepath.Join(destDir, filename)
+	if err := os.MkdirAll(filepath.Dir(outputPath), 0755); err != nil {
+		return e.Wrapf(err, "error creating directory for file %q", filename)
+	}
+	return e.Wrap(os.WriteFile(outputPath, []byte(content), os.FileMode(0644)), "error writing file")
+}
+
+// SetupPath checks if possible to set the output path
+func SetupPath(path string, removedOutputPathFlag bool) error {
```

Comment:
Here I have the same feelings as for `WriteFile` - no idea what `npguardutils.SetupPath` does without reading the code.

Reply:
reverted